bindinfo/handle.go

```diff
@@ -408,6 +408,91 @@ func (h *BindHandle) DropBindRecord(originalSQL, db string, binding *Binding) (e
 	return err
 }
 
+// SetBindRecordStatus set a BindRecord's status to the storage and bind cache.
+func (h *BindHandle) SetBindRecordStatus(originalSQL, db string, binding *Binding, newStatus string) (err error) {
+	db = strings.ToLower(db)
+	h.bindInfo.Lock()
+	h.sctx.Lock()
+	defer func() {
+		h.sctx.Unlock()
+		h.bindInfo.Unlock()
+	}()
+	exec, _ := h.sctx.Context.(sqlexec.SQLExecutor)
+	_, err = exec.ExecuteInternal(context.TODO(), "BEGIN PESSIMISTIC")
+	if err != nil {
+		return err
+	}
+	var (
+		updateTs               types.Time
+		oldStatus0, oldStatus1 string
+		affectRows             int
+	)
+	if newStatus == Disabled {
+		// For compatibility reasons, when we need to 'set binding disabled for <stmt>',
+		// we need to consider both the 'enabled' and 'using' status.
+		oldStatus0 = Using
+		oldStatus1 = Enabled
+	} else if newStatus == Enabled {
+		// In order to unify the code, two identical old statuses are set.
+		oldStatus0 = Disabled
+		oldStatus1 = Disabled
+	}
+	defer func() {
+		if err != nil {
+			_, err1 := exec.ExecuteInternal(context.TODO(), "ROLLBACK")
+			terror.Log(err1)
+			return
+		}
+
+		_, err = exec.ExecuteInternal(context.TODO(), "COMMIT")
+		if err != nil || affectRows == 0 {
+			if affectRows == 0 {
+				logutil.BgLogger().Warn("[sql-bind] There are no bindings can be set the status")
```

Comment:
How about returning this warning message to users instead of logging it to notice them to check whether they input the SQL and binding correctly.